.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
technically you should not have this here. 

try to gitignore

Reply:
i see you ignored this

however i fliped a coin and decided to forgive you

consider this resolved